to-do.list

```diff
@@ -4,3 +4,4 @@ Track project milestone and track deadline
 Participate in team meeting by 1 pm to discuss project progress
 Collaborate in training sessions for the new software 
 Report to team head
+You can resign as well.
```

Comment:
Hi Adetayo

Thanks for the review
Line 7 seems to have error,  kindly look into it, thanks.